lib/Serialization/Serialization.cpp

```diff
@@ -3142,6 +3142,9 @@ void Serializer::writeDecl(const Decl *D) {
 
     auto conformances = theClass->getLocalConformances(
                           ConformanceLookupKind::All, nullptr);
+    llvm::erase_if(conformances, [](const ProtocolConformance *conf) {
```

Comment:
Should this be a new ConformanceLookupKind?

Reply:
Maybe. This is a quick-and-dirty test to see if it breaks things, without really thinking through the implications.